### PR TITLE
test

### PR DESCRIPTION
**Pull request by anniehedgpeth**
## Background

Related: #245 
This branch seeks to fix the authentication issues to the TFC backend.

### CODE REVIEW

File: .github/workflows/handler-test.yml [hunk 1]
```diff
@@ -1020,43 +980,28 @@ jobs:
...
       - name: Write Terraform Variables
         working-directory: ${{ env.WORK_DIR_PATH }}
-        env:
-          GOOGLE_SERVICE_ACCOUNT: ${{ secrets.GOOGLE_SERVICE_ACCOUNT }}
-          TFE_HOSTNAME: ${{ secrets.TFE_HOSTNAME }}
-          TFE_ORGANIZATION: ${{ secrets.TFE_ORGANIZATION }}
-          TFE_TOKEN: ${{ secrets.TFE_TOKEN }}
-          TFE_WORKSPACE: ${{ secrets.TFE_WORKSPACE }}
         run: |
           ip_address=$( dig +short @resolver1.opendns.com myip.opendns.com )
```
Comment (p0pr0ck5) [inline .github/workflows/handler-test.yml]: is this needed anymore?

File: .github/workflows/handler-test.yml [hunk 2]
```diff
@@ -795,43 +772,28 @@ jobs:
...
```
Comment (p0pr0ck5) [inline .github/workflows/handler-test.yml]: is this needed anymore?

**Review by p0pr0ck5 — APPROVED** [review]

**Review by jkerry — APPROVED** [review]

**Pull request merged**